index.html

```diff
@@ -13,82 +13,76 @@
-                    <a href='#location'>Location</a>
+                    <a href='#location' data-i18n="step1.title"></a>
                 </div>
                 <div class='col4 center pad1 step-2'>
-                    <a>Details</a>
+                    <a href='#details' data-i18n="step2.title"></a>
                 </div>
                 <div class='col4 center pad1 step-3'>
-                    <strong>Done</strong>
+                    <strong data-i18n="step3.title"></strong>
                 </div>
             </div>
 
             <div class='col12' id='address-step'>
                 <div class='center pad2'>
-                    <span id='instructions'>Start by telling us where your business is located.</span>
+                    <span id='instructions' data-i18n="step1.desc"></span>
+                    <p id="couldnt-find" style='display: none;' data-i18n="[html]step1.fail"></p>
                 </div>
                 <div class='col12'>
                     <div class='pad1'>
                         <form id='find'>
-                            <input class='col8' id='address' placeholder='501 West Main Street, Alexandria, VA' />
-                            <button class='col4' id='findme'>Find Me!</button>
+                            <input class='col8' id='address' placeholder='' data-i18n="[placeholder]step1.addr"/>
+                            <button class='col4' id='findme' data-i18n="step1.button"></button>
                         </form>
                     </div>
                     <br />
                     <br />
                     <div class='col12' id='findme-map'></div>
-                    <p id="couldnt-find" style='display: none;'>
-                        <strong>We couldn't find your address.</strong> Try searching for your street or city without the address.
-                    </p>
                 </div>
             </div>
 
             <div class='col12 hide' id='collect-data-step'>
-                <p>We found your location. Now tell us about your business.</p>
+                <p data-i18n="[html]step2.desc"></p>
                 <div class='col12'>
-                    <div class='col12'>
-                        <label for='category'>Category</label>
-                        <select id='category' data-placeholder='Pick a category...' ></select>
+                    <div class='clearfix col12'>
+                        <label for='category' data-i18n="step2.catlabel"></label>
+                        <input id='category' data-placeholder='' class='col10' data-i18n="[data-placeholder]step2.catplaceholder"/>
                     </div>
-                    <div>
-                        <label for='name'>Name</label>
-                        <input id='name' type='text' placeholder='ACME Parts' />
-                        <label for='phone'>Phone Number</label>
-                        <input id='phone' type='text' placeholder='414-555-1212' />
+                    <div class='col6'>
```

Comment:
Ah, I was worried it was my PR that tweaked the layout. Will redo the improve-metadata PR now that i18n is in